docs/source/DeveloperGuide/WebAppDIRAC/DevelopWebApp/index.rst

```diff
@@ -27,119 +27,148 @@ Each application consists of two parts:
 
 The folder structure of the server side web installation is as follows:
 
-* <Module name folder such as DIRAC, LHCbDIRAC, WebAppDIRAC>
-
-   * **WebApp**
-      * __init__.py
-      * **handler**: contains all the server side implementations of the framework and all the applications.
-         * __init__.py
-      * **static**: contains all the static content that can be loaded by the client side such as JavaScript files, images and css files
-         * <Module name folder such as DIRAC, LHCbDIRAC, WebAppDIRAC>: contains the client side implementation of each application
-            * Application 1
-            * Application 2
-      * **template**: contains all the templates used by the files in the handler folder
+::
+
+  <Module name such as DIRAC, LHCbDIRAC, WebAppDIRAC>
+  └── WebApp
+      ├── __init__.py
+      ├── handler
+      │   # Server side implementations of the framework and all the applications
+      │   └── __init__.py
+      ├── static
+      │   # Content that can be loaded by the client side such as JavaScript files, images and css files
+      │   └── <Module name such as DIRAC, LHCbDIRAC, WebAppDIRAC>
+      │       # Client side implementation of each application
+      │       ├── Application
```

Comment:
Application[12] are subfolders of DIRAC/LHCbDIRAC, no? so there should be another of the `|_` ?